README.md

```diff
@@ -4,14 +4,14 @@
   <img src="./src/Open_Grants_Program.png" style="width:1300px";>
 </p>
 
-**:loudspeaker: Important:** The Web3 Foundation _Open Grants_ Program focuses on technology grants up to **USD 30k for initial grants** and **USD 100k for follow-up grants**, which are tracked **transparently** on GitHub and disbursed in **Bitcoin or DAI**.
```

Comment:
I would leave the accentuation because it makes clear there are two programs. It's too easy to glance over the name of the program and not everyone might know.

Reply:
agree